WorkingSet/WorkingSet.cpp

```diff
@@ -0,0 +1,335 @@
+#include "inc/WorkingSet.h"
+#include <algorithm>
+#include <indicators/progress_bar.hpp>
+#include <iostream>
```

Comment:
I might be confused here. Please justify.

Reply:
are you talking about the indicators include? iostream? if you are, iostream is necessary to output the anti cursor character and indicators is helpful when the dynamic sets are being generated

Reply:
The question here is why are you using a progress bar here? Are you reading a trace and my method is insufficient? Is your code just that slow?

Reply:
it takes that long. When processing the address time space for non-trivial programs, this can become a significant part of the runtime. For example, 
brisk: ~140s for trace pass, ~700s for address time space
2DConv: ~52s for trace pass, ~30s for address time space
MatrixMultiply: ~47m for trace pass, ~20m for address time space